cmd/mount.go

```diff
@@ -218,6 +218,15 @@ func mount(c *cli.Context) error {
 		}
 	}()
 	installHandler(mp)
+
+	mntLabels := prometheus.Labels{
+		"vol_name": format.Name,
+		"mp":       mp,
+	}
+	// Wrap the default registry, all prometheus.MustRegister() calls should be afterwards
```

Comment:
Also  `prometheus.Register`

blob = object.WithMetrics(blob)

Reply:
OK, fix in next commit.